src/models/User.ts

```diff
@@ -1,9 +1,10 @@
-import { Network } from './Network';
-import type { UserGroup } from './UserGroup';
+interface RemoteGatewayIds {
+  [key: string]: object;
```

Comment:
can the types be more specific

```suggestion
  [key: Node['id']]: User;  // maybe User is wrong, but use a more specific type instead of object
```